static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -298,6 +298,14 @@ define([
                 if (!($slot.hasClass('ad-slot--top-above-nav') && size === '1,1')) {
                     $slot.parent().css('display', 'block');
                 }
+
+                if (($slot.hasClass('ad-slot--top-banner-ad') && size === '88,70')) {
+                    $slot.addClass('ad-slot__fluid250');
+                }
+
+                if (($slot.hasClass('ad-slot--commercial-component') && size === '88,88')) {
+                    $slot.addClass('ad-slot__fluid250');
+                }
```

Comment:
You can combine it into one condition eg:

if (($slot.hasClass('ad-slot--top-banner-ad') && size === '88,70')) || ($slot.hasClass('ad-slot--commercial-component') && size === '88,88')) {
   $slot.addClass('ad-slot__fluid250');
}